narratives/ncov_sit-rep_2020-05-15.md

```diff
@@ -0,0 +1,220 @@
+licenseLink: "https://creativecommons.org/licenses/by/4.0/"
+dataset: "https://nextstrain.org/ncov/global/2020-05-14?d=tree&legend=closed"
+
+abstract: "Nextstrain Situation Reports use publicly shared genomic data to track the spread and evolution of SARS-CoV-2. This week, we give an overview of viral mutations and what they mean (and don't mean) for the COVID-19 pandemic."
+---
+<!-- Translators: Only text after : in the above ^ needs to be translated -->
+<!-- Comment tags like these do not need to be translated, they are only to help you! -->
+<!-- Ensure that links always end in a 'letter' (. counts) If some kind of text doesn't follow them, it breaks the slide. -->
+<!-- numbers can be tagged ilke this: 161</tag> - this is just for us to help find them to update! Just leave in the </tag> bit. -->
+
+<!-- This is left-side text -->
+# [Executive summary](https://nextstrain.org/ncov/2020-05-14?d=tree,entropy&p=grid)
+
+We analyzed 5,193 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how SARS-CoV-2 is evolving and moving around the world. This week, we focus on viral mutations. We cover:
+<br><br>
+* Where viral mutations come from (they're normal)  
+* The rate of mutation in SARS-CoV-2 (very typical)
+* How many strains of SARS-CoV-2 are circulating (as far as we know: 1)  
+* How geography and epidemiology contribute to perceived differences in viral genotypes (this is tricky)
+
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text 2-->
+# [COVID-19 Resources](https://nextstrain.org/ncov/global/2020-05-14?d=tree&p=full&legend=closed)
+
+#### Nextstrain Resources
+* [START HERE: How to read a phylogeny](https://nextstrain.org/narratives/trees-background/).
+* [Previous Situation Reports](https://nextstrain.org/ncov-sit-reps/).
+* [Explanatory Twitter threads](https://bedford.io/misc/twitter/).
+* [Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).
+* [Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11).
+
+#### External Resources
+* [How coronavirus mutations and spreads (NYTimes)](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html).
+* [Ask a Scientist & FAQs](https://covid19.fas.org/l/en).
+* [WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).
+* [CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).
+
+<!-- This is the right-side text -->
+```auspiceMainDisplayMarkdown
+
+# Situation Report Hiatus
+<p>
+Early in the pandemic, it was unclear how SARS-CoV-2 was moving between countries, where it was circulating locally, and how localized outbreaks were related.
+Over the last four months, we've tackled these questions and more in our weekly Situation Reports. At this stage of the pandemic, it's clear that these broad-strokes patterns are largely consistent across regions and countries:  
```

Comment:
```suggestion
Over the last four months, we've tackled these questions - and more - in our weekly Situation Reports. At this stage of the pandemic, it's clear that these broad-strokes patterns are largely consistent across regions and countries:  
```
Just to clarify sentence structure. I read it super awkwardly first time: "we've tackled these questions" / "and more in our weekly Situation Reports"